source/agora/node/Ledger.d

```diff
@@ -835,15 +875,18 @@ version (unittest)
                 ? params_
                 : (blocks.length > 0
                    // Use the provided Genesis block
-                   ? new immutable(ConsensusParams)(cast(immutable)blocks[0], WK.Keys.CommonsBudget.address)
+                   ? new immutable(ConsensusParams)(cast(immutable)blocks[0], WK.Keys.CommonsBudget.address,
+                            1008, 7, 80, 30, 1, 1024, 200, 600)
                    // Use the unittest genesis block
-                   : new immutable(ConsensusParams)());
+                   : new immutable(ConsensusParams)(1008, 7, 80, 30, 1, 1024, 200, 600));
```

Comment:
I could most probably got away with specifying 600 for this second constructor here, but I think it is debatable whether the 2 constructors of the ConsensusParams should have different defaults(even the second constructor which is exclusively used by tests might make more sense with 60 default)